source/sphinx_extensions/plugin_directory/templates/action.rst

```diff
@@ -39,8 +39,9 @@
        <h4>Import:</h4>
 
 .. command-block::
+   :no-exec:
 
-       from {{import_path}} import {{action_api_name}}
+   from {{import_path}} import {{action_api_name}}
 
```

Comment:
Can you drop the blank line below the import? @ChrisKeefe & I were just looking at this - it adds a newline to the copy-to-clipboard button.

Reply:
Removing the newline does not seem to fix the issue (also all of our directives are formatted with an extra newline to separate the directive from the rest of the content).

Reply:
Bummer.